codegens/python-requests/lib/python-requests.js

```diff
@@ -110,7 +110,6 @@ self = module.exports = {
       ', data = payload, files = files' : ', data = payload';
     snippet += !options.followRedirect ? ', allow_redirects=False' : '';
```

Comment:
Leave space behind and after `=` in `allow_redirects=False` in generated python code. as we are doing the same in other places. i.e `data = payload` and `files = files`.